pss/pss.go

```diff
@@ -112,11 +112,23 @@ func (p *Pss) Register(topic trojan.Topic, hndlr Handler) {
 
 // Deliver allows unwrapping a chunk as a trojan message and calling its handler func based on its topic
 func (p *Pss) Deliver(c chunk.Chunk) {
-	m, _ := trojan.Unwrap(c)
-	h := p.getHandler(m.Topic)
-	if h != nil {
-		h(*m)
+	if isPotentialTrojan(c) {
```

Comment:
this should just be part of trojan package.

Reply:
done